apps/wallet-dashboard/components/Dialogs/Staking/views/ConfirmAndExit.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Button,
+    ButtonType,
+    Card,
+    CardBody,
+    CardImage,
+    CardType,
+    Header,
+    ImageShape,
+    ImageType,
+} from '@iota/apps-ui-kit';
+import { ValidatorApyData } from '@iota/core';
+import { Validator } from './Validator';
+import { IotaLogoMark } from '@iota/ui-icons';
+import { StakingTransactionDetails } from './StakingTransactionDetails';
+import { Layout, LayoutBody, LayoutFooter } from './Layout';
+
+interface SuccessScreenViewProps {
+    validatorAddress: string;
+    gasBudget: string | number | null | undefined;
+    onConfirm: () => void;
+    amount: string;
+    symbol: string | undefined;
+    validatorApy: ValidatorApyData;
+    onClose: () => void;
+}
+
+export function SuccessScreenView({
+    validatorAddress,
+    onConfirm,
+    amount,
+    symbol,
+    onClose,
+    validatorApy: { apy, isApyApproxZero },
+    gasBudget,
+}: SuccessScreenViewProps): React.JSX.Element {
+    return (
+        <Layout>
+            <Header title="Transaction" onClose={onClose} />
+            <LayoutBody>
+                <div className="flex flex-col gap-y-md">
+                    <Validator address={validatorAddress} isSelected showAction={false} />
+
+                    <Card type={CardType.Outlined}>
+                        <CardImage type={ImageType.BgSolid} shape={ImageShape.Rounded}>
+                            <IotaLogoMark className="h-5 w-5 text-neutral-10" />
+                        </CardImage>
+                        <CardBody title={`${amount} ${symbol}`} subtitle="Stake" />
+                    </Card>
+
+                    <StakingTransactionDetails
+                        apy={apy}
+                        isApyApproxZero={isApyApproxZero}
+                        gasBudget={gasBudget}
+                    />
+                </div>
+            </LayoutBody>
+
+            <LayoutFooter>
+                <div className="flex w-full">
+                    <Button
+                        type={ButtonType.Primary}
+                        fullWidth
+                        onClick={onConfirm}
+                        text="Confirm & Exit"
```

Comment:
at this point, the stake is already created, so I think the button should simply say "Finish," as there's nothing left to confirm :see_no_evil: just a thought to improve ux :pray:

Reply:
It was different in the figma when I did it :triumph:

Reply:
yes, actually in figma it's 'Confirm & Exit,' but I feel that it's clearer with 'Finish' :pray:

Reply:
Well they changed it now

Reply:
honestly, I think it's the other way around :see_no_evil:  should trust our instincts in situations like this, not figma

Reply:
but that might be a problem, imagine i change something because I feel another way it's correct, we might be blocking the PR because of changes that not everybody likes. Just an example, not what it's happening here

Reply:
if the change were a big deal, I’d totally agree with you, but I’m suggesting this because it’s a small thing, and we have it like this in the wallet too